outputs.tf

```
@@ -1,4 +1,3 @@
-output "admin_account_id" {
-  description = "The AWS account ID of the AWS Firewall Manager administrator account."
-  value       = join("", aws_fms_admin_account.default.*.id)
+output "aws_fms_account" {
```

Wouldn't `admin_account` be more specific?
```suggestion
output "admin_account" {
```